PullToRefresh/PullToRefresh.swift

```diff
@@ -47,7 +47,7 @@ public class PullToRefresh: NSObject {
     
     // MARK: - State
     
-    var state: State = .Initial {
+    public private(set) var state: State = .Initial {
```

Comment:
In our component it's not used, but someone could create a subclass and override the property